### ISSUE TITLE
[question] is there any way to query an specific callsign using an endpoint?

### ISSUE THREAD

**Issue by g3nsvrv** [nguyenquyhy/Flight-Tracker#18]
[question] is there any way to query an specific callsign using an endpoint?, e.g. https://events.flighttracker.tech/?ownAircraft=callsign

**Comment by nguyenquyhy** [comment]
Yes https://events.flighttracker.tech/?myCallsign=HY-308

**Comment by g3nsvrv** [comment]
@nguyenquyhy wow, that was fast and amazing!, thank you so much!!!

**Comment by nguyenquyhy** [comment]
There are several others too (follow, show route, show plan). I use those to use the map as an overlay in OBS. If you would like to describe more on your usecase, I might be able to provide some more.

**Comment by g3nsvrv** [comment]
@nguyenquyhy yeah, that would be helpful, I think would be amazing to have like a reference documentation about those endpoints and the different options available.

For me, I was only looking for a "shareable link" to my flight, but as you said, it could be used to almost anything, you can develop a bot on twitter or somewhere else when a callsign is on the map, or something similar.

**Comment by nguyenquyhy** [comment]
I was actually trying to create a page where you can click and choose what you want and it generates the URL. But my time spread a bit too thin among all the plugins I have so still haven't completed that page yet. Probably I'll try to add a document some time this week.

**Comment by g3nsvrv** [comment]
many thanks @nguyenquyhy !!!
